src/mock.rs

```diff
@@ -0,0 +1,151 @@
+use crate::expectation::model::{Expectation, HttpMethod};
+
+pub struct IncomingRequest {
+    pub method: HttpMethod,
+    pub code: i32,
```

Comment:
Don't understand this `code` exists.
it seems to modelise http code but it concern only response so not `IncomingRequest`
Can you explain this code field ? :pray: